scapy/arch/pcapdnet.py

```diff
@@ -16,9 +16,307 @@
 from scapy.supersocket import SuperSocket
 from scapy.error import Scapy_Exception
 import scapy.arch
+import socket
 
 
+if conf.use_winpcapy:
+  #mostly code from https://github.com/phaethon/scapy translated to python2.X
```

Comment:
I think that it is better to give the full link to the commit/file that was used.